### ISSUE TITLE
APIv1: Bookmarks

### ISSUE THREAD

**Issue by fmartingr** [go-shiori/shiori#659]

**Comment by fmartingr** [comment]
> Hi. I don't know how you want to impeliment API for adding bookmark but if it possible please return something that we can distinguish between internal error and bookmark exist. Currently shiori return status 500 if bookmark exist.

Yeah, on successful creation the response would be `201` and on duplicated `409`.

**Comment by towerstreet** [comment]
How to use this API to add bookmark？
